### PR TITLE
Improve Light Theme Consistency and CSS Annotations

### PR DESCRIPTION
**Pull request by gauravuj**
## Summary
This PR introduces several improvements to the light theme to ensure it aligns with the current product style. Additionally, we've enhanced the CSS file with detailed annotations for better readability and understanding.

## Changes
- Adjusted light theme colors to match the latest product style guide.
- Annotated CSS properties and values throughout the `light-theme.css` file for clarity.
- Optimized CSS selectors for improved performance.

## Impact
These changes will provide a more cohesive and accessible user experience across the light theme, while also making it easier for developers to understand and maintain the styling code.

## Screenshots
Screenshot 2024-03-27 at 2 41 36 PM

### CODE REVIEW

**Comment by cyqjoseph** [comment]
Good stuff, but maybe fix the checkstyle issues first then we can merge!

File: src/main/java/seedu/address/ui/CommandListPanel.java [hunk 1]
```diff
@@ -45,15 +48,17 @@ private void loadCommands() {
                 new CommandInfo("view", "Views all consultations"),
                 new CommandInfo("update", "Updates a consultation"),
                 new CommandInfo("cancel", "Cancels a consultation"),
-                new CommandInfo("search", "Search a consultation")
+                new CommandInfo("search", "Search a consultation"),
+                new CommandInfo("theme", "Changes the current theme")
```
Comment (cyqjoseph) [inline src/main/java/seedu/address/ui/CommandListPanel.java:52]: A much needed addition!

File: src/test/java/seedu/address/model/command/CommandInfoTest.java
Comment (cyqjoseph) [inline src/test/java/seedu/address/model/command/CommandInfoTest.java:1]: Good coverage!

**Review by cyqjoseph — APPROVED** [review]
Overall LGTM, just need to clear up some minor checkstyle issues!

File: src/main/java/seedu/address/ui/CommandListPanel.java (hunk 1 above)
Comment (jerremyng) [reply src/main/java/seedu/address/ui/CommandListPanel.java:52]: + 1

**Review by jerremyng — APPROVED** [review]
LGTM, thanks for adding javadocs and theme improvements!

**Comment by ChrysanthemumT** [comment]
I don't really see the need for the extra panel?

File: src/test/java/seedu/address/model/command/CommandInfoTest.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+import org.junit.jupiter.api.Test;
+
+public class CommandInfoTest {
+
+    @Test
+    public void constructor_validCommandInfo_creationSuccessful() {
+        // Setup
```
Comment (ChrysanthemumT) [inline src/test/java/seedu/address/model/command/CommandInfoTest.java:12]: don't think these comments are necessary, should limit comments that don't add value

**Review by ChrysanthemumT — COMMENTED** [review]
looks good overall, but the extra panel is weird

**Pull request merged**